source/Sashimi/CloudTemplates/TerraformHclCloudTemplateHandler.cs

```diff
@@ -49,13 +49,11 @@ public Metadata ParseTypes(string template)
         public object ParseModel(string template)
         {
             var parameters = GetVariables(template);
-            return parameters?
-                   .Select(x => new KeyValuePair<string, object?>(
-                                                                  x.Value,
-                                                                  GetDefaultValue(x))
-                          )
-                   .ToDictionary(x => x.Key, x => x.Value)
-                   ?? new Dictionary<string, object?>();
+
+            if (parameters != null)
+                return parameters.Select(x => new KeyValuePair<string, object?>(x.Value, GetDefaultValue(x)))
```

Comment:
I assume this is part-and-parcel of the net5 update?

Reply:
the net5 compiler is different about how it matches up parameters - it was erroring here.